tests/cli/test_cmds_spark_run.py

```diff
@@ -1374,6 +1405,8 @@ def test_paasta_spark_run_pyspark(
         needs_docker_cfg=False,
         aws_region="test-region",
         force_spark_resource_configs=False,
+        use_eks=False,
```

Comment:
I don't think we have test-case for use_eks=True, right?
If not, can you add one?